lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -302,11 +302,6 @@ export async function getReleaseNotesMd(
   const { baseUrl, repository } = project;
   const version = release.version;
   logger.trace(`getReleaseNotesMd(${repository}, ${version})`);
-  const skippedRepos = ['facebook/react-native'];
```

Comment:
this only skips the changelog file, but not the changes from releases. so this skip is intentional. 

why should it be removed? I think we should extend this to known projects which have both too.

@rarkins do you remember why it was added in first place?

Reply:
I couldn't find why it was originally added - I did go through every comment last PR to touch this LOC, but couldn't spot anything obvious.

Does it have anything to do with the CHANGELOG MDs being split across three files? 🤔 

It feels like a variant of this which exists for GitHub sourcing https://github.com/renovatebot/renovate/blob/eef6f171f20a4c310bd50f99d5c881ca2ff46634/lib/workers/repository/update/pr/changelog/github/source.ts#L28-L37

Reply:
It was added long ago: https://github.com/renovatebot/renovate/commit/8f6c2393839ec4c333a520a08463efa1431d8d29

Looks like today we'd be better off using the changelog rather than releases?

Reply:
@setchy can you provide a screenshot of before and after with this change?

Reply:
Sure thing. 

I've created the following reproduction repository, with two PRs and also pushed the PR body MD to that it's easier to diff

**Current behavior (main branch)**
* https://github.com/setchy/renovate-react-native/pull/2
* https://github.com/setchy/renovate-react-native/blob/main/PR-BODY.md

**Refactor PR behavior**
* https://github.com/setchy/renovate-react-native/pull/4
* https://github.com/setchy/renovate-react-native/blob/main/PR-BODY-REFACTOR.md

I plan to go through these in more detail shortly...

Reply:
It appears to have made the changelog much longer. Normally that would be for the best, but it seems *really* long

Reply:
The other way around actually. 

main-branch PR body
* 939 lines
refactor-branch PR body
* 376 lines

One main difference of note, the (v0.72.0 changelog md entry](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0720)  is HUUGGGEEE and is being partially truncated.  Their [release-notes equivalent](https://github.com/facebook/react-native/releases/tag/v0.72.0) is succinct.

Reply:
Confirmed that it's the truncated in alignment with the logic in https://github.com/renovatebot/renovate/blob/0bc7e61089f5069119d56c1e5cf9bf3af4afbea8/lib/modules/platform/github/index.ts#L1741

Reply:
Having said this, I think the special case handling for react-native makes sense to keep, mainly due to the way they split their CHANGLOG.md into 3 files (2 are previous release archives).


My plan is to introduce a more generic set of functions within [changelog/sources](https://github.com/renovatebot/renovate/blob/0bc7e61089f5069119d56c1e5cf9bf3af4afbea8/lib/workers/repository/update/pr/changelog/source.ts) to handle the different `skip` types
* `shouldSkipPackage` - existing behavior
* `shouldSkipChangelogMd` - will add and implement for react-native
* `shouldSkipReleaseNotes` - for future